metadata/src/main/java/org/apache/kafka/controller/QuorumController.java

```diff
@@ -1635,7 +1636,7 @@ private enum ImbalanceSchedule {
     /**
      * Tracks if a snapshot generate was scheduled.
      */
-    private boolean generateSnapshotScheduled = false;
+    private final boolean generateSnapshotScheduled;
```

Comment:
If the value is always set to `false`, why have the variable at all? 🤔

Reply:
> If the value is always set to `false`, why have the variable at all? 🤔

Hi, I also want to directly delete this variable because it has not been used anywhere, but I am not sure if this variable will be used in subsequent iteration development.

Reply:
It seems that this variable has been deleted in # 13540 and can be directly ignored...